Library/Formula/tig.rb

```diff
@@ -43,10 +43,18 @@ class Tig < Formula
   def install
     system "./autogen.sh" if build.head?
     system "./configure", "--prefix=#{prefix}", "--sysconfdir=#{etc}"
-    system "make", "install"
+    system "make", "install", "sysconfdir=#{prefix/"share/tig/examples"}"
```

Comment:
Yes, you are right. `make install` will have to be done in a separate step. I'll try to find time to update the patch against the latest version and see if that fixes the build failure.